syncd/syncd_flex_counter.h

```diff
@@ -82,9 +87,9 @@ class FlexCounter
             std::vector<sai_port_stat_t> portCounterIds;
         };
 
-        FlexCounter(uint32_t pollInterval);
-        static FlexCounter& getInstance(uint32_t pollInterval);
-        static void removeInstance(uint32_t pollInterval);
+        FlexCounter(std::string instanceId);
```

Comment:
>instanceId [](start = 32, length = 10)

can you explain this instanceId concept? I am not familiar with it.

Reply:
It looks like the instanceId is the flex counter group name, so maybe change to groupname is more clear?

---
In reply to: [166061744](https://github.com/Azure/sonic-sairedis/pull/289#discussion_r166061744) [](ancestors = 166061744)

Reply:
Each group will be an instance in flex counter and instance id is the group name.